runtime/src/runtime.rs

```diff
@@ -482,10 +494,7 @@ impl Runtime {
 
   fn push_first_child_in_stack(&mut self, settings: &BlockSettings) -> bool {
     match self.get_block(settings.children[0]) {
-      cuentitos_common::Block::Text { id: _, settings: _ } => {
-        self.push_stack(settings.children[0]);
-        true
-      }
+      cuentitos_common::Block::Text { id: _, settings: _ } => self.push_stack(settings.children[0]),
       cuentitos_common::Block::Choice { id: _, settings: _ } => {
         println!("Make a choice");
```

Comment:
This should not be part of the runtime.
This should probably be an error of some sort.

Reply:
And this string should just be part of the CLI.